src/udev/udev-rules.c

```diff
@@ -1876,8 +1904,12 @@ static int udev_rule_apply_token_to_event(
         }
         case TK_M_IMPORT_PARENT: {
                 char buf[UDEV_PATH_SIZE];
+                bool truncated;
+
+                (void) udev_event_apply_format(event, token->value, buf, sizeof(buf), false, &truncated);
+                if (truncated)
+                        log_rule_warning(dev, rules, "Result of substitution into %s is truncated.", token->value);
```

Comment:
The message is also misleading. Because it's the name of the thing to import that is truncated, not the thing itself.

And importing the wrong thing because the name got truncated seems very iffy…